hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/utils/VersionInfo.java

```diff
@@ -85,8 +85,12 @@ public String getSrcChecksum() {
     return info.getProperty("srcChecksum", "Unknown");
   }
 
-  public String getHadoopProtocVersion() {
```

Comment:
This class is marked `Public` and `Stable`, I guess we should keep this method for compatibility.

Reply:
done